packages/renderer/src/lib/task-manager/ProgressBar.svelte

```diff
@@ -29,12 +29,19 @@
 </style>
 
 <script lang="ts">
-export let progress: number | undefined;
-export let width: string = 'w-36';
-export let height: string = 'h-4';
+import type { HTMLAttributes } from 'svelte/elements';
+
+interface Props extends HTMLAttributes<HTMLElement> {
+  progress?: number;
+  width?: string;
+  height?: string;
+  class?: string;
+}
+
+let { progress, width = 'w-36', height = 'h-4', class: className, ...restProps }: Props = $props();
```

Comment:
```suggestion
let { progress, width = 'w-36', height = 'h-4', ...restProps }: Props = $props();
```

Reply:
I would keep class outside `restProps` so we do not overwrite the existing class attribute by spreading to the parent div